include/nlohmann/detail/meta/type_traits.hpp

```diff
@@ -157,6 +157,9 @@ template<class B1, class... Bn>
 struct conjunction<B1, Bn...>
 : std::conditional<bool(B1::value), conjunction<Bn...>, B1>::type {};
 
+// https://en.cppreference.com/w/cpp/types/negation
+template<class B> struct negation : std::integral_constant<bool, !B::value> { };
```

Comment:
Wouldn't this be rather `negation_v`?

Reply:
`negation` is the base struct, `negation_v` is 

    template<class B>
    inline constexpr bool negation_v = negation<B>::value;

Reply:
Ah - I read the `::value` at the wrong place. Thanks!